### PR TITLE
Switch module path to Bayer-Group

### PR DESCRIPTION
**Pull request by poost-frey**

### CODE REVIEW

File: reporters.go [hunk 1]
```diff
@@ -6,19 +6,19 @@ import (
...
+			realArgTypes[i] = fmt.Sprintf("...%v", toFriendlyName(functionType.In(i).Elem()))
 			continue
 		}
 
-		real[i] = toFriendlyName(functionType.In(i))
+		realArgTypes[i] = toFriendlyName(functionType.In(i))
 	}
 
-	testReporter.Errorf("mocka: expected arguments of type (%v), but received (%v)", strings.Join(real, ", "), strings.Join(mapToTypeName(arguments), ", "))
+	testReporter.Errorf("mocka: expected arguments of type (%v), but received (%v)", strings.Join(realArgTypes, ", "), strings.Join(mapToTypeName(arguments), ", "))
```
Comment (cmarfia) [inline reporters.go:21]: I don't see the value in this type of change. This one and the one below it are small enough that they will not cause any other side effects so we can keep them. However, I would like to keep changes to mocka small and focused. That way we don't introduce bugs or anything by making unnecessary changes like renames.

**Review by cmarfia — APPROVED** [review]

File: reporters.go (hunk 1 above)
Comment (poost-frey) [reply reporters.go:21]: fair enough. this was an issue uncovered by the linter which is why I did it as part of this PR
